src/app/features/room/RoomViewHeader.tsx

```diff
@@ -324,8 +344,28 @@ export function RoomViewHeader() {
             )}
           </Box>
         </Box>
+
         <Box shrink="No">
-          {!ecryptedRoom && (
+          {false && isDirectMessage() && (
+            <TooltipProvider
+              position="Bottom"
+              align="End"
+              offset={4}
+              tooltip={
+                <Tooltip>
+                  <Text>Start a Call</Text>
+                </Tooltip>
+              }
+            >
+              {(triggerRef) => (
+                <IconButton onClick={handleCall} ref={triggerRef}>
+                  <Icon size="400" src={Icons.Phone} />
+                </IconButton>
+              )}
+            </TooltipProvider>
+          )}
+
+          {!ecryptedRoom && !room.isCallRoom() && (
```

Comment:
Make Search and Pin buttons visible in call rooms if chat is open
```suggestion
          {!ecryptedRoom && (!room.isCallRoom() || isChatOpen) && (
```